brianlong/www.validators.app
# 181922231 add eversol pool

#### What's this PR do?
- add new stake pool and check if stats are correct

#### How should this be manually tested?
- go to /stake-pools and checkout the new pool
- check if all stats seems legit
- check if there's a logo of the new pool on landing page

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/181922231)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
